added net8.0 to target frameworks. updated test project to net8.0

VILLAN3LL3 opened the pull request.

VILLAN3LL3 committed:
updated project to net8.0

VILLAN3LL3 committed:
re-added netstandard 1.3

uhaciogullari commented:
Hi @VILLAN3LL3 

Thanks for your pull request. I'm not actively working on this project or even with .NET at the moment. Feel free to publish your own fork.

VILLAN3LL3 committed:
use v8.0.0 for Npgsql

renovate[bot] committed:
Add renovate.json

VILLAN3LL3 committed:
Merge pull request #1 from VILLAN3LL3/renovate/configure

Configure Renovate

renovate[bot] committed:
Update dependency Microsoft.Extensions.Configuration to v8

VILLAN3LL3 committed:
Create dotnet.yml

VILLAN3LL3 committed:
Update dotnet-desktop.yml

VILLAN3LL3 committed:
Merge pull request #3 from VILLAN3LL3/renovate/major-dotnet-monorepo

Update dependency Microsoft.Extensions.Configuration to v8

VILLAN3LL3 committed:
[Snyk] Security upgrade Npgsql from 3.2.7 to 4.0.14 (#6)

* fix: src/DoomedDatabases.Postgres/DoomedDatabases.Postgres.csproj to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-DOTNET-NPGSQL-6825563

* removed netstandard 1.3

* test adaption

* try running test with postgres db

---------

Co-authored-by: snyk-bot <snyk-bot@snyk.io>

renovate[bot] committed:
Update dependency Npgsql to 8.0.3 (#5)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

VILLAN3LL3 committed:
added package readme

VILLAN3LL3 committed:
updated package information

VILLAN3LL3 commented:
> Hi @VILLAN3LL3
> 
> Thanks for your pull request. I'm not actively working on this project or even with .NET at the moment. Feel free to publish your own fork.

https://www.nuget.org/packages/VILLAN3LL3.DoomedDatabases.Postgres/1.0.0

VILLAN3LL3 committed:
added action to publish to nuget

VILLAN3LL3 committed:
added inputs

VILLAN3LL3 committed:
Update README.md

VILLAN3LL3 committed:
Update README.md

VILLAN3LL3 committed:
Update README.md
